.gitignore

```diff
@@ -227,4 +227,5 @@ pip-log.txt
 #############
 
 composer.lock
-vendor
\ No newline at end of file
+vendor
+/logs
```

Comment:
No, log folder location is a personal setting. I usually keep it in the same folder as the project for convenience, so i don't have to switch between applications when checking the logs file. I just keep a tab open in php storm. So let's treat it like we treat the ignores for our personal IDE helper folders (.idea or .settings for eclipse, for example).